bin/bootstrap.sh

```diff
@@ -39,11 +39,11 @@ REPO_DIR=$(mktemp -d)
 cp -a . "${REPO_DIR}"
 cd "${REPO_DIR}" || die "Failed to cd to ${REPO_DIR}"
 
-for file in $(git grep --files-with-matches replace-this-with-the-track-slug); do
-    sed -i "s/replace-this-with-the-track-slug/${SLUG}/g" "${file}"
-done
+mapfile -t files < <(git grep --files-with-matches replace-this-with-the-track-slug)
+sed -i "s/replace-this-with-the-track-slug/${SLUG}/g" "${files[@]}"
```

Comment:
On second thought, xargs might be better:
```suggestion
git grep --null --files-with-matches replace-this-with-the-track-slug) \
| xargs --null sed -i "s/replace-this-with-the-track-slug/${SLUG}/g"
```

Reply:
I'm not generally a huge fan of xargs. Is there anything wrong with passing the files to `sed`? I really don't think we need to protect against newlines in the file names here.

Reply:
and it's very unlikely we'll find thousands of filenames.